pkg/redis/redis.go

```diff
@@ -0,0 +1,49 @@
+package redis
+
+import (
+	"errors"
+	"time"
+
+	"github.com/go-redis/redis"
+)
+
+type IRedis interface {
+	Initialise() *redis.Client
+	StoreKeyValue(client *redis.Client, key string, value string, expiration time.Duration)
```

Comment:
why pass in a `redis.Client` to an inteface.It makes our interface implicitly coupled with the `go-redis` package